pixelgl/window.go

```diff
@@ -424,3 +424,8 @@ func (w *Window) Clear(c color.Color) {
 func (w *Window) Color(at pixel.Vec) pixel.RGBA {
 	return w.canvas.Color(at)
 }
+
+// GetCanvas returns the window's underlying Canvas
```

Comment:
I don't think this method is needed. If someone wants to use a shader, they can always just create a canvas and then draw it on a window.

Reply:
I would like to have access to the underlying window canvas, but maybe there are good reasons not to expose it.

Reply:
It doesn't hurt to have the method right? Gives freedom to the user and it's only 1 method so it doesn't pollute the API :)

Reply:
I'd prefer to drop the `Get` prefix though.